wallet/wallet-controller/src/read.rs

```diff
@@ -264,7 +264,9 @@ impl<'a, T: NodeInterface> ReadOnlyController<'a, T> {
         tasks.try_collect().await
     }
 
-    pub fn get_created_blocks(&self) -> Result<Vec<Id<GenBlock>>, ControllerError<T>> {
+    pub fn get_created_blocks(
+        &self,
+    ) -> Result<Vec<(BlockHeight, Id<GenBlock>)>, ControllerError<T>> {
```

Comment:
Can we move `wallet_rpc_lib::types::BlockInfo` into the controller crate and use that instead of the tuple?

Reply:
moved